src/model/Model.lua

```diff
@@ -25,20 +25,20 @@ function Model:new()
     self.__index = self
     self.model = {}
     self.sectionNameMap = {
-         ["r"] = "request_definition",
-         ["p"] = "policy_definition",
-         ["g"] = "role_definition",
-         ["e"] = "policy_effect",
-         ["m"] = "matchers"
+		["r"] = "request_definition",
```

Comment:
I see there are tabs used like this in some places, please replace them with normal 4-spaces.